### PR TITLE
Tuning the resource requests/limits to fit in smaller namespaces.

### PR DESCRIPTION
**Pull request by caggles**

### CODE REVIEW

File: crunchydb/high-availablility/PostgresCluster.yaml [hunk 1]
```diff
@@ -5,13 +5,39 @@ metadata:
 spec:
   monitoring:
     pgmonitor:
+      # this stuff is for the "exporter" container in the "hippo-ha-pgha1" set of pods
       exporter:
         image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi8-5.0.4-0
```
Comment (matthieu-foucault) [inline crunchydb/high-availablility/PostgresCluster.yaml]: Any `image` tag should be removed and left to the operator's default. Setting an image may lead to incompatibilities when updating the operator

File: crunchydb/high-availablility/PostgresCluster.yaml [hunk 2]
```diff
@@ -5,13 +5,39 @@ metadata:
...
+        resources:
+          requests:
+            cpu: 50m
+            memory: 64Mi
+          limits:
+            cpu: 100m
+            memory: 256Mi
   image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-13.5-0
   postgresVersion: 13
```
Comment (matthieu-foucault) [inline crunchydb/high-availablility/PostgresCluster.yaml:25]: do we want to set the default to `14`?

File: crunchydb/high-availablility/PostgresCluster.yaml [hunk 3]
```diff
@@ -5,13 +5,39 @@ metadata:
 spec:
   monitoring:
```
Comment (matthieu-foucault) [inline crunchydb/high-availablility/PostgresCluster.yaml]: This entire block is only useful if pg monitor is deployed. We should leave a comment about that

File: crunchydb/high-availablility/PostgresCluster.yaml [hunk 4]
```diff
@@ -5,13 +5,39 @@ metadata:
...
+          resources:
+            requests:
+              cpu: 50m
+              memory: 32Mi
+            limits:
+              cpu: 100m
+              memory: 64Mi
       dataVolumeClaimSpec:
         accessModes:
         - "ReadWriteOnce"
```
Comment (matthieu-foucault) [inline crunchydb/high-availablility/PostgresCluster.yaml:51]: The storage request below should be lowered to fit in the extra-small namespaces

File: crunchydb/high-availablility/PostgresCluster.yaml (hunk 2 above)
Comment (caggles) [reply crunchydb/high-availablility/PostgresCluster.yaml:25]: i'm not particularly picky about this - this is meant to be a template more than anything else, so users are free to update it on their own!

**Review by StevenBarre — APPROVED** [review]
:shipit:

**Review by matthieu-foucault — APPROVED** [review]

**Pull request merged**